docs/rest_api.md

```diff
@@ -240,7 +240,7 @@ default channel, the VLAN drivers choose `vlan/native`.
 
 `GET /networks`
 
-List all networks.
+List all networks or list all public networks.
```

Comment:
I still think the way this is structured is a little clumsy; the information about when you get which operation is *sortof* implicit in the authorization requirements, but it's still a little weird in that it's not actually the description of the operation itself, and it doesn't technically say how you get one operation or another, just the access you need.

I'd add a short paragraph that spells out how the decision is made.

Reply:
Sure, make sense, will give it a try